### PR TITLE
PP-12123: try sending metrics from deploy-adminusers

### PR DESCRIPTION
**Pull request by jfharden**
Considering we have hundreds of places to send the metrics I've put a lot of effort in deploy-to-test to be able to parse all the different kinds of inputs and generate app names and release numbers, and to use yaml anchors and loaded variables so we can not have to repeat literally thousands of lines of code.

I've run all the jobs and have created a simple grafana dashboard which shows the release number sent at each step:

https://grafana.monitoring.pay-cd.deploy.payments.service.gov.uk/d/e5000e04-deef-4ac7-8ffc-b7364c241dd1/release-pipeline-releases?orgId=1
Screenshot 2024-02-16 at 17 29 28

### CODE REVIEW

File: ci/pipelines/deploy-to-test.yml [hunk 1]
```diff
@@ -157,7 +319,7 @@ resources:
     icon: github
     source:
       uri: https://github.com/alphagov/pay-ci
-      branch: master
+      branch: pp-12123/send-release-metric-for-adminusers-deploy
```
Comment (rfisher-gds) [inline ci/pipelines/deploy-to-test.yml:322]: Branch needs changing prior to merge.

**Review by rfisher-gds — CHANGES_REQUESTED** [review]
Looks good, and I think the anchors are appropriately granular. Just needs the branch putting back to `master`.

File: ci/pipelines/deploy-to-test.yml (hunk 1 above)
Comment (jfharden) [reply ci/pipelines/deploy-to-test.yml:322]: Good call!

Comment (jfharden) [reply ci/pipelines/deploy-to-test.yml]: Updated

**Review by rfisher-gds — APPROVED** [review]

**Pull request merged**